libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -244,6 +244,11 @@ class AP_AHRS {
     void get_relative_position_D_home(float &posD) const;
     bool get_relative_position_D_origin(float &posD) const WARN_IF_UNUSED;
 
+    // return location corresponding to vector relative to the
+    // vehicle's origin
```

Comment:
"vehicle's origin" seems new.. I guess this is the EKF origin?

Reply:
I think maybe we should rename these to "get_location_from_origin_offset()" and "get_location_from_home_offset()"

Reply:
> "vehicle's origin" seems new.. I guess this is the EKF origin?

Yes; thing is that it might not be an EKF that your origin-relative-position is being supplied by - and, indeed, our EKFs now shift their origin....  This could probably be "AHRS origin"...

Reply:
> I think maybe we should rename these to "get_location_from_origin_offset()" and "get_location_from_home_offset()"

Done.